config/vault.go

```diff
@@ -310,8 +328,9 @@ func (c *VaultConfig) GoString() string {
 		"Token:%t, "+
 		"VaultAgentTokenFile:%t, "+
 		"Transport:%#v, "+
-		"UnwrapToken:%s"+
+		"UnwrapToken:%s, "+
```

Comment:
Drive-by formatting fix. Thanks!

Reply:
I was trying to figure out why the unit tests would not pass, and I noticed this typo when my print statements were all weird.